### PR TITLE
update_template_config

### PR DESCRIPTION
**Pull request by hunght**
Changes
 added pangea_dapp to package

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -20,5 +20,12 @@
   "dependencies": {
     "pangea-sdk": "git+https://github.com/Bit-Nation/pangea-sdk.git#e1575021ad2c17f168a258e68c24d467661bf606",
     "truffle": "^4.1.13"
+  },
+  "pangea_dapp": {
+    "name": {
+      "en-us": "send / request money"
```
Comment (florianlenz) [inline package.json]: Rename this from `send / request money` to `DApp Name`

File: package.json [hunk 2]
```diff
@@ -20,5 +20,12 @@
...
+    },
+    "engine": "1.0.7",
```
Comment (florianlenz) [inline package.json]: Change the version to `0.1.0`

**Review by florianlenz — CHANGES_REQUESTED** [review]

**Comment by florianlenz** [comment]
and remove the old config please ...

File: package.json (hunk 2 above)
Comment (hunght) [reply package.json]: done

File: package.json (hunk 1 above)
Comment (hunght) [reply package.json]: done

**Review by florianlenz — APPROVED** [review]

**Pull request merged**